webpack/index.js

```diff
@@ -0,0 +1,26 @@
+require('expose?$!expose?jQuery!jquery');
+require('jquery-ujs');
+require('expose?_!lodash');
+require('expose?jstz!jstz');
+require('expose?ipaddr!ipaddr.js');
+require('jquery.cookie');
+require('expose?JsDiff!diff');
+require('./modules/bundle_flot');
```

Comment:
I dont think webpack default lookup path is a solution for repeating strings. If so it can get huge really fast